dotnet/src/SemanticKernel/Memory/MemoryRecord.cs

```diff
@@ -110,28 +116,45 @@ public class MemoryRecord : DataEntryBase
     /// Create a memory record from a serialized metadata string.
     /// </summary>
     /// <param name="json">Json string representing a memory record's metadata.</param>
-    /// <param name="embedding">The embedding associated with a memory record.</param>
-    /// <param name="key">Optional existing database key</param>
-    /// <param name="timestamp">optional timestamp</param>
+    /// <param name="embedding">Optional embedding associated with a memory record.</param>
+    /// <param name="key">Optional existing database key.</param>
+    /// <param name="timestamp">optional timestamp.</param>
     /// <returns></returns>
     /// <exception cref="MemoryException"></exception>
     public static MemoryRecord FromJson(
         string json,
-        Embedding<float> embedding,
+        Embedding<float>? embedding,
```

Comment:
Shouldn't this also be the json-encoded (string) embedding? Or are we keeping those separate?

Reply:
The embedding should be separate from the metadata - perhaps I should rename this to FromJsonMetadata. Constructing a memory record from existing memories should be embedding optional.

Reply:
renamed to FromJsonMetadata to make more clear